pkg/sql/subscriber.go

```diff
@@ -191,23 +199,36 @@ func (s *Subscriber) consume(ctx context.Context, topic string, out chan *messag
 
 		messageUUID, noMsg, err := s.query(ctx, topic, out, logger)
 		if err != nil {
+			errorsInARow += 1
 			if isDeadlock(err) {
 				logger.Debug("Deadlock during querying message, trying again", watermill.LogFields{
 					"err":          err.Error(),
 					"message_uuid": messageUUID,
 				})
 			} else {
-				logger.Error("Error querying for message", err, watermill.LogFields{
-					"wait_time": s.config.RetryInterval,
-				})
+				if errorsInARow >= s.config.ErrorLogThreshold {
+					logger.Error("Error querying for message, will retry", err, watermill.LogFields{
+						"wait_time":    s.config.RetryInterval,
+						"errorsInARow": errorsInARow,
```

Comment:
Perhaps we could use `errors_in_a_row` just to be consistent with snake_case in field names?

Also worth considering if we could deduplicate the message and fields a bit maybe.